client/src/components/Input/Input.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components/macro";
+
+const Inputfield = styled.input`
+  height: 40px;
+  width: 330px;
+  border: solid 2px var(--secondary-color);
+  background: none;
+  color: var(--inputfield-color);
+  font-family: "Heiti SC", adobe-heiti-std, sans-serif;
+  font-size: 14px;
+  display: flex;
+  text-align: center;
+`;
+
+export const Input = ({ title }) => {
+  return <Inputfield placeholder={title} />;
+};
+
+Input.propTypes = {
+  title: PropTypes.string.isRequired,
+};
```

Comment:
You can (and maybe should) directly export the styled component.
It is confusing that `title` equals `placeholder`, because the HTML `title` attribute is something different.
Don't create new names for existing properties.

```suggestion
import styled from "styled-components/macro";

export const Input = styled.input`
  height: 40px;
  width: 330px;
  border: solid 2px var(--secondary-color);
  background: none;
  color: var(--inputfield-color);
  font-family: "Heiti SC", adobe-heiti-std, sans-serif;
  font-size: 14px;
  display: flex;
  text-align: center;
`;
```

In addition, you have to update your stories and set the `placeholder`.